api/open_saves.pb.go

```diff
@@ -15,16 +15,16 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.27.1
-// 	protoc        v3.12.4
+// 	protoc        v3.19.4
```

Comment:
Can we move these proto changes to another PR?

Reply:
I simply removed these changes.